pkgs/applications/networking/pjsip/default.nix

```diff
@@ -33,19 +36,36 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  nativeBuildInputs =
+    lib.optional pythonSupport [ swig ];
+
   buildInputs = [ openssl libsamplerate ]
+    ++ lib.optional pythonSupport python3
```

Comment:
This needs to be in `nativeBuildInputs` otherwise the `pkgsCross.aarch64-multiplatform` variant fails.

Otherwise `nixpkgs-review` is happy, nixos x86_64. `python3Packages.pjsua2` is importable as a python module.

Reply:
Aah cool, yes, python as nativeBuildInput makes sense. Adding it to `nativeBuildInputs` did work partly, but `${python.interpreter}` still referred to the non-native python binary, which failed the build. I reverted `${python.interpreter}` to `python` again, that (combined with python in nativeBuildInputs`) did work to run: `nix build .#pkgsCross.aarch64-multiplatform.pjsip`

I understand the output of nixpkgs-review a bit better now as well, that resulted in:

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>jami-client-qt</li>
    <li>ring-daemon</li>
  </ul>
</details>
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>jami-client</li>
    <li>jami-daemon</li>
    <li>pjsip</li>
    <li>python310Packages.pjsua2</li>
    <li>python311Packages.pjsua2</li>
  </ul>
</details>